cedar-policy-validator/src/cedar_schema/fmt.rs

```diff
@@ -27,12 +27,61 @@ use thiserror::Error;
 use crate::{json_schema, RawName};
 use cedar_policy_core::{ast::InternalName, impl_diagnostic_from_method_on_nonempty_field};
 
+/// Number of spaces of indentation per level in the Cedarschema file
+pub const NUM_INDENTATION_SPACES: usize = 2;
+
+/// Support for formatting with a given base indentation (spaces) that should be applied after newlines.
+trait IndentedDisplay {
+    fn fmt_indented(
+        &self,
+        f: &mut std::fmt::Formatter<'_>,
+        base_indentation: usize,
+    ) -> std::fmt::Result;
+}
+
+/// Display Annotations (from the est package) with a given base indentation
+struct IndentedAnnotations<'a>(&'a cedar_policy_core::est::Annotations, usize);
+
+impl Display for IndentedAnnotations<'_> {
```

Comment:
Happy to go into `cedar_policy_core::est::Annotations` and implement `IndentedDisplay` there too. However, then we need to find a generic place to put `IndentedDisplay`...

Reply:
I did this for now, let's see if we need this more generally later. I avoided cedar-policy-core depending on these schema-specific indentation types for now by keeping core as simple as possible (just `usize`)